client/scss/components/_buttons.scss

```diff
@@ -0,0 +1,74 @@
+@mixin secondary {
+  background-color: color('transparent');
+  border-width: 2px;
+
+  &:not([disabled]):hover,
+  &:not([disabled]):focus {
+    background-color: color('transparent');
+  }
+}
+
+.btn {
+  @include font('HNM5');
+  display: inline-block;
+  width: 100%;
+  min-width: 160px;
+  min-height: 42px;
+  padding: 10px 30px;
+  color: color('white');
+  background-color: color('action-green-1');
+  border: 1px solid color('action-green-1');
+  border-radius: 6px;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
```

Comment:
Only for light text on dark backgrounds. Have just removed from the buttons with dark text